pkg/client/unversioned/clientcmd/config.go

```diff
@@ -75,8 +77,16 @@ func (o *PathOptions) GetLoadingPrecedence() []string {
 	if envVarFiles := o.GetEnvVarFiles(); len(envVarFiles) > 0 {
 		return envVarFiles
 	}
-
-	return []string{o.GlobalFile}
+	files, err := loadFileOrDir(o.GlobalFile)
+	// If GlobalFile doesn't exist, try GlobalFileDir
+	if os.IsNotExist(err) {
+		files, err = loadFileOrDir(o.GlobalFileDir)
+	}
+	if err != nil {
+		glog.Warningf("Failed to load config file(s): %v", err)
```

Comment:
In the CLI we should only use either `glog` with level (`glog.V(n).`) or plain `fmt`.